src/Kiota.Builder/Writers/TypeScript/CodeClassDeclarationWriter.cs

```diff
@@ -21,7 +21,11 @@ public override void WriteCodeElement(CodeClass.Declaration codeElement, Languag
                                                                     .GroupBy(x => x.Item3)
                                                                     .OrderBy(x => x.Key);
             foreach (var codeUsing in importSymbolsAndPaths)
-                writer.WriteLine($"import {{{codeUsing.Select(x => GetAliasedSymbol(x.Item1, x.Item2)).Distinct().OrderBy(x => x).Aggregate((x,y) => x + ", " + y)}}} from '{codeUsing.Key}';");
+                if (!string.IsNullOrWhiteSpace(codeUsing.Key))
```

Comment:
since the blank import issue is fixed, could you revert that change please?

Reply:
This is valid check which would help avoiding empty imports completely. I had encountered the problem with empty imports before as well.